index.html

```diff
@@ -1,62 +1,101 @@
 <section>
-    <h3 id = "text">Experience:</h3>
-    <ul>
-        <li id ="experience">Wake Tech. Community College AAS Major Web Technologies/ Web Design: 2014- 2018 </li>
-        <li id ="experience">Red Rover Bootcamp: March 2023- May 2023</li>
-        <li id ="experience">Code The Dream: April 2023- current</li>
-    </ul>
+    <h2 id = "text">Experience:</h2>
+    <div class="grid-container">
+        <div id = "experience">Code The Dream: April 2023- current</div>
+        <div id = "experience">Red Rover Bootcamp:March 2023- May 2023</div>
+        <div id = "experience">Wake Tech. Community College: AAS Major Web Technologies/ Web Design: 2014- 2018</div>
+    </div>
 </section>
 <section id = "skills">
     <h2 id = "text">Skills</h2>
     <ul class = "skillsBox">
     </ul>
 </section>
 <section>
-    <h3 id = "text">Connect:</h3>
+    <h2 id = "text">Connect:</h2>
     <ul id = "connect">
-        <li><strong>Gmail:</strong> fetter.oksana@gmail.com</li>
-        <li><strong>GitHub:</strong><a href = " https://github.com/ofeterovskaya"> https://github.com/ofeterovskaya </a></li>
-        <li><strong>Twitter:</strong><a href = " https://twitter.com/OksanaFetter"> https://twitter.com/OksanaFetter </a></li>
-        <li><strong>LinkedIn:</strong><a href = " https://www.linkedin.com/in/oksana-feterovskaya-037587138/"> https://www.linkedin.com/in/oksana-feterovskaya-037587138/ </a></li>
+        <li><strong>Gmail:</strong> fetter.oksana@gmail.com </li>
+        <li><strong>GitHub:</strong><a href = " https://github.com/ofeterovskaya" target = "_blank"> https://github.com/ofeterovskaya </a></li>
+        <li><strong>Twitter:</strong><a href = " https://twitter.com/OksanaFetter" target = "_blank"> https://twitter.com/OksanaFetter </a></li>
+        <li><strong>LinkedIn:</strong><a href = " https://www.linkedin.com/in/oksana-feterovskaya-037587138/" target = "_blank"> https://www.linkedin.com/in/oksana-feterovskaya-037587138/ </a></li>
     </ul>
 </section>
 <section id = "messagesForm">
     <h2 id = "text">Leave a Message</h2>
     <form name ="leave_message">
-        <label for ="name"> Name </label><br>
-            <input type = "text" name = "usersName" required = "true" ><br>
-        <label for = "email"> Email </label><br>
-            <input type = "email" name = "usersEmail" required = "true" ><br>
-        <label for = "message">Message</label><br>
+        <div class="flex-container">
+            <label class="flex-name" for ="name"> Name </label><br>
+                <input type = "text" name = "usersName" required = "true" ><br>
+            <label class="flex-email" for = "email"> Email </label><br>
+                <input type = "email" name = "usersEmail" required = "true" ><br>
+            
+        <label class="flex-message" for = "message">Message</label><br>
             <textarea name ="usersMessage" required = "true" ></textarea><br>
-        <button type = "submit">Submit</button>
+        </div>
```

Comment:
Fix: Labels `for` attributes should refer to an `id` on a form input and instead of just pointing to the `name` attribute.

For example:
```html
<label for="usersName">Your name</label>
<input type="text" id="usersName" name="usersName" required />
```
You may be wondering... why do I need to include a `name` attribute then? 
> The name attribute is used to reference elements in JavaScript or to reference form data after a form is submitted.
>
> Only form elements with a **name** attribute will have their values passed when submitting a form.

Having the label refer to an `id` also aids in auto-complete results:
![](https://github.com/butolinka/fenrir-intro/assets/22607722/d983b803-d718-4d34-bd0b-2f74acce617c)